### ISSUE TITLE
Main supply cache on the mini map [Improvement Proposal]

### ISSUE THREAD

**Issue by DHFan45** [DarklightGames/DarkestHour#1566]
### Is your proposal related to a problem?

Almost every map has an "Main supply cache" which is also physically visible to the player. Unlike the standard "supply cache", however, the "Main supply cache" is not displayed on the "mini map".

What do you think about displaying the "Main supply cache" on the "mini map" as well?

The symbol could be the same as the standard "supply cache" but in a different color.
This would make it much easier for the player to find and navigate to the "Main supply cache", especially on some maps it is hard to find in the terrain or the main spawn point is not located at the "Main supply cache".

![1ance](https://user-images.githubusercontent.com/123426586/218279819-5d926bde-869a-4cf6-b68b-1dea48aefa56.png)

### Describe the solution you'd like

x

**Comment by LordDz** [comment]
Yes please, I've sometimes placed down an ammo crate next to it just to remember where the main supply is as it's often not placed directly by the spawn point.
